projects/python-client/src/datapane/client/api/report/processors.py

```diff
@@ -502,6 +522,55 @@ def _open_server(host: str, port: int) -> None:
         webbrowser.open_new_tab(f"http://{host}:{port}")
 
 
+class Stringify(LocalProcessor):
+    """
+    Stringifies a given App as a single HTML string
+    """
+
+    served = False
+    template_name = "template.html"
```

Comment:
should `template_name` be a boolean, i.e. will there be other templates down the line?

```python
use_ipython_template: bool = False
```

Reply:
The plan is to allow for other templates down the line! In `processors.py` we are concerned with stringifying in general, as the feature now exists regardless of IPython integration!